packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/translations/Messages_fr_FR.json

```diff
@@ -1,6 +1,19 @@
 {
-  "ip_agora_table_header_offer": "Offre",
+  "ip_agora_table_header_offer": "Choisissez votre offre",
```

Comment:
some time we use imperative verb and sometimes infinitive verb.
For for consistency, i guess we have to use one of both forms:
`Choisissez` or `Choisisser`